src/vibrato.ts

```diff
@@ -61,29 +58,34 @@ export class Vibrato extends Modifier {
 
     this.position = Modifier.Position.RIGHT;
     this.renderOptions = {
-      harsh: false,
-      vibratoWidth: 20,
-      waveHeight: 6,
-      waveWidth: 4,
-      waveGirth: 2,
+      code: 0xeab0,
+      width: 20,
     };
 
-    this.setVibratoWidth(this.renderOptions.vibratoWidth);
-  }
-
-  /** Set harsh vibrato. */
-  setHarsh(harsh: boolean): this {
-    this.renderOptions.harsh = harsh;
-    return this;
+    this.setVibratoWidth(this.renderOptions.width);
   }
 
   /** Set vibrato width in pixels. */
   setVibratoWidth(width: number): this {
-    this.renderOptions.vibratoWidth = width;
-    this.setWidth(width);
+    this.renderOptions.width = width;
+    this.text = String.fromCodePoint(this.renderOptions.code);
+    this.measureText();
+
+    const items = Math.round(this.renderOptions.width / this.getWidth());
+    for (let i = 1; i < items; i++) {
+      this.text += String.fromCodePoint(this.renderOptions.code);
+    }
+    this.measureText();
+
     return this;
   }
 
+  /** Set vibrato code. */
+  setVibratoCode(code: number): this {
```

Comment:
It is not clear what codes are supported. We should point developers to https://w3c.github.io/smufl/latest/tables/multi-segment-lines.html

And teach the developer if they want a:

```
U+EAB5
wiggleWavy
Wavy line segment
```

Then they need to pass in the code `0xeab5`.

Reply:
Maybe we need a block comment at the top of the file with a list of the recommended codes and their friendly SMuFL names. Just reading the code with these hex code points is confusing. :-)